supplements/dnd-beyond/monstrous-compendium-2.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<elements>
+	<info>
+		<update version="0.0.1">
+			<file name="monstrous-compendium-2.xml" url="https://raw.githubusercontent.com/AuroraLegacy/elements/master/supplements/dnd-beyond/monstrous-compendium-2.xml" />
+		</update>
+	</info>
+
+	<element name="Monstrous Compendium Volume Two: Dragonlance Monsters" type="Source" source="Core" id="ID_WOTC_SOURCE_MONSTROUS_COMPENDIUM_TWO">
+		<description>
+			<p></p>
+		</description>
+		<setters>
+			<set name="abbreviation">MCV2</set>
+			<set name="url">https://www.dndbeyond.com/claim/source/mcv2</set>
+			<set name="image">https://raw.githubusercontent.com/AuroraLegacy/elements/master/supplements/dnd-beyond/MCV2.png</set>
+			<set name="author" abbreviation="WOTC" url="http://dnd.wizards.com">Wizards of the Coast, D&amp;D Beyond</set>
+			<set name="official">true</set>
+			<set name="supplement">true</set>
+			<set name="release">20221205</set>
+		</setters>
+	</element>
+	<element name="Nightbringer" type="Magic Item" source="Monstrous Compendium Volume Two: Dragonlance Monsters" id="ID_WOTC_MCV2_MAGIC_ITEM_NIGHTBRINGER">
+		<description>
+			<p>This heavy mace has a dark oaken handle and a head of blackened steel trimmed with gold. In combat, the mace’s head is wreathed in inky black energy.</p>
+			<p class="indent">You gain a +3 bonus to attack and damage rolls made with this weapon. When you hit with an attack roll using it, the target takes an additional 4d4 radiant damage.</p>
+			<p class="indent">If you have an evil alignment, you also gain the following benefits when attuned to Nightbringer:</p>
+			<p class="indent"><strong><em>Blessing of the Dragon Queen.</em></strong> You are immune to the charmed and frightened conditions, and you gain immunity to one of the following damage types (choose when you attune to the weapon): acid, cold, fire, lightning, or poison.</p>
+			<p class="indent"><strong><em>Eye of Midnight.</em></strong> You have darkvision with a range of 60 feet. If you already have darkvision, its range increases by 60 feet.</p>
+			<p class="indent"><strong><em>Midnight Shroud.</em></strong> The weapon has 6 charges. When you hit a creature with an attack using this weapon, you can expend one charge to force the creature to make a DC 20 Constitution saving throw. On a failed save, the creature is blinded until the start of your next turn. The weapon regains 1d6 expended charges daily at dawn.</p>
+		</description>
+		<setters>
+			<set name="keywords">mace, evil</set>
+			<set name="category">Magic Weapons</set>
+			<set name="cost" currency="gp">0</set>
+			<set name="type" addition="mace">Weapon</set>
+			<set name="attunement">true</set>
+			<set name="rarity">Legendary</set>
+			<set name="enhancement">3</set>
+			<set name="weapon">ID_INTERNAL_WEAPON_GROUP_MACES</set>
+		</setters>
+		<rules>
```

Comment:
The second part of Blessing of the Dragon Queen where the player can choose a damage type to gain immunity to has not been implemented

Reply:
Should those be implemented as a class feature selection?